ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.72.yaml

```diff
@@ -0,0 +1,367 @@
+base_images:
+  eks:
+    name: automation-flavors
+    namespace: stackrox
+    tag: eks-0.3.1
+  golang-1.17.2:
+    name: golang
+    namespace: stackrox
+    tag: 1.17.2
```

Comment:
Isn't this a bit too old? We run on 1.17.12 or 13 now, right?

Reply:
That's the version used in nightlies. Do we want to bump it now for the release?

Reply:
Just spotted this, so not sure how much it affects the release. Nightlies on the release branch sound strange. Isn't it the builder that would build the upstream released version?

Reply:
Let's maybe ask @gavin-stackrox what this version mens to learn for the future.

The PR is good to be merged for me 👍